### PR TITLE
Added flexbox to the cards for responsive cards. Also updated some of…

### PR DESCRIPTION
**Pull request by raregamer**
… the text styling for the cards. I Added some greek gods. Dependencies changed to react-native-web. I had to change the dependency for react-native-web to work, for me. This is also my first time ever using react native, I followed docs and previous code as my guide.

### CODE REVIEW

**Comment by vardhanapoorv** [comment]
@raregamer how did you test your changes? I appreciate you giving react-native a try!

**Comment by raregamer** [comment]
Hi thanks, I am new to testing. I am looking into working and testing it with Jest right now. If you have any suggestions on what I should checkout to help me test it properly let me know.

**Comment by vardhanapoorv** [comment]
@raregamer I mean to ask how did you check your changes? Running which npm script ?
Yes, jest is a good option.

**Comment by raregamer** [comment]
I used expo start --web

**Comment by raregamer** [comment]
Hi let me know if there any adjustments I need to make or things I should check out.

File: react-native/greek-mythology/package.json [hunk 1]
```diff
@@ -11,7 +11,6 @@
     "expo": "^35.0.0",
     "react": "16.8.3",
     "react-dom": "16.8.3",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
```
Comment (vardhanapoorv) [inline react-native/greek-mythology/package.json]: Don't remove this since it wasn't working without it for me.

**Comment by vardhanapoorv** [comment]
@raregamer good job!

**Comment by raregamer** [comment]
Awesome thank you!

**Pull request merged**